From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 13 Dec 2024 09:06:30 +0100
Subject: soc: qcom: pd_mapper: Add SM7225 compatible

The Qualcomm SM7225 is practically identical to SM6350, so add an entry
using that data.

Signed-off-by: Luca Weiss <luca.weiss@fairphone.com>
---
I double checked and the Android builds for this platform also have a
.jsn file for the equivalent of &mpss_root_pd_gps but this is not part
of sm6350_domains upstream.

But I actually can't find any evidence of this actually being used in
the AMSS sources. Just some Android proprietary user space seems to try
and find 'gps/gps_service' but nothing relevant for 'msm/modem/root_pd'

Should we still correct this? Everything appears to work as expected and
"#define DEBUG" in the driver also doesn't show anything being wrong.
---
 drivers/soc/qcom/qcom_pd_mapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/qcom/qcom_pd_mapper.c b/drivers/soc/qcom/qcom_pd_mapper.c
index 6e30f08..d98738f 100644
--- a/drivers/soc/qcom/qcom_pd_mapper.c
+++ b/drivers/soc/qcom/qcom_pd_mapper.c
@@ -553,6 +553,7 @@ static const struct of_device_id qcom_pdm_domains[] __maybe_unused = {
 	{ .compatible = "qcom,sm4250", .data = sm6115_domains, },
 	{ .compatible = "qcom,sm6115", .data = sm6115_domains, },
 	{ .compatible = "qcom,sm6350", .data = sm6350_domains, },
+	{ .compatible = "qcom,sm7225", .data = sm6350_domains, },
 	{ .compatible = "qcom,sm7325", .data = sc7280_domains, },
 	{ .compatible = "qcom,sm8150", .data = sm8150_domains, },
 	{ .compatible = "qcom,sm8250", .data = sm8250_domains, },
